suzieq/engines/pandas/engineobj.py

```diff
@@ -178,12 +181,8 @@ def get_valid_df(self, table: str, **kwargs) -> pd.DataFrame:
         for fld in key_fields:
             if fld not in fields+addnl_fields:
                 addnl_fields.insert(0, fld)
-                drop_cols.append(fld)
 
-        for f in addnl_fields:
-            if f not in fields:
-                # timestamp is always the last field
-                fields.insert(-1, f)
+        getcols = set(fields+addnl_fields)
```

Comment:
This should be a list
```suggestion
        getcols = list(set(fields+addnl_fields))
```

Reply:
I know, but does it matter?